README.md

```diff
@@ -118,6 +118,22 @@ Code linting is performed using:
 [Bullet](https://github.com/flyerhzm/bullet) runs around each spec. If it
 detects an N+1 query it will raise an exception and the tests will fail.
 
+### Creating data migrations
+
+When running a live service sometimes you're required to change existing data in
+some way. Using the gem [data_migrate](https://github.com/ilyakatz/data-migrate)
+we can write data migrations in a similar way we do with schema migrations.
+
+The migrations are stored in the `db/data` folder.
+
+- To generate a migration: `rails g data_migration add_this_to_that`
+- To run the data migration by itself: `rails data:migrate`
+- To run the data migration with the schema migration:
+  `rails db:migrate:with_data`
+
+When the service deploys it will run these data migrations at the same time as
+the schema ones.
```

Comment:
can we reword this – if we run `rails db:migrate` followed by `rails data:migrate` sequentially rather than using `rails db:migrate:with_data`, which looks like it intermingles the two. I'd rather we had a workflow that required schema migrations were run first and then data migrations run after. that does some changes will require splitting across more than one release (i.e. a follow up PR to drop the now-unused column), but I think this approach is safer and less likely to result in irreversible accidental data loss.

Reply:
Yeah that makes sense, I've made the change